apps/server/lib/server/code_reloader.ex

```diff
@@ -1,14 +1,9 @@
 defmodule Server.CodeReloader do
   def reload!(_, _) do
-    # This transaction ensures that this function is only called once during the code reload.
-    # This change was due to an error where the web app.js would not load properly and web event handlers were broken
-    :global.trans({:resource_id, :lock_requester_id}, fn ->
-      Mix.in_install_project(fn ->
-        # Fixed in future Hex, but needed for now
-        _ = Application.ensure_all_started(:hex)
-        Mix.Task.rerun("deps.compile", ["kino_live_view_native"])
-        Mix.Task.clear()
-      end)
-    end)
+    # Calling `Mix.Project.app_path()` when Phoenix from Livebook produces the following error:
+    # Cannot access build without an application name, please ensure you are in a directory with a mix.exs file and it defines an :app name under the project configuration
+    # This function is called by `Mix.Project.consolidation_paths/1` when recompiling because `Mix.Project.umbrella?/1` returns false.
+    # This custom code reloader hides the error, though it will still print in the Livebook console, but not in Livebook code cells.
+    :ok
```

Comment:
The LOCs that you removed did exactly that. I don't think those should have been removed. As it stands this PR doesn't actually fix anything.

Reply:
That's correct.

The reloader code I removed raises an error. However, I have that code working on a newly scaffolded umbrella application, and a non-umbrella application.

I'm currently identifying why a new umbrella app, and non umbrella app both work but the `kino_live_view_native` project doesn't.

Reply:
I forget the reason why we went with an umbrella app in the first place, do you recall the context? If there's no good reason for it you can go with a mono app.

Reply:
I don't think there was a specific reason. I'll work on making this a mono app since that has a few additional benefits
* we can version releases on hex
* I believe we'll get slightly faster build times

Reply:
I've pushed up the changes to make this library a mono application.